### ISSUE TITLE
Defs not working

### ISSUE THREAD

**Issue by Yeriss** [Lambda11/let-me-fish#23]
Keep getting this error in my command box when I try to !fish

[mods] ERROR: Module "let-me-fish-master" uses the following outdated/unsupported packets:
[mods] ERROR: - C_END_FISHING_MINIGAME.1
[mods] ERROR: - C_RQ_ADD_ITEM_TO_DECOMPOSITION_CONTRACT.1
[mods] ERROR: - C_START_FISHING_MINIGAME.1
[mods] ERROR: - S_FISHING_BITE.1
[mods] ERROR: - S_START_FISHING_MINIGAME.1
[mods] ERROR: Please contact the module's author: https://github.com/Lambda11/let-me-fish/issues

Grabbed fresh download of mod, still not working. Issue with mod or w/my proxy? I use Caali's.

**Comment by Owyn** [comment]
It seems like you haven't put def \ opcode files into correct location as the installation instruction says

**Comment by Yeriss** [comment]
Thanks, I was able to get it working. I forgot to move those files when I moved to my laptop instead of desktop.

So I'm not sure whats causing it but I'm now having dismantling problems. It will dismantle about half the fish and then stop and say you are out of that item and then start back up fishing with my inventory very cluttered full of fish still. Also the fish gold command will give the "gold fish dismantling enabled" return but it won't actually dismantle the gold fish. Any ideas whats causing that? Like if I misplaced a file somewhere that might have caused it? Thanks.
